compose/rails.md

```diff
@@ -53,6 +53,7 @@ to link them together and expose the web app's port.
         image: postgres
       web:
         build: .
+        command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
```

Comment:
My 2c:
Isn't this more obscure / difficult to read? I can see the value of having something that "just works", but I'm concerned this might be taking away from the educational value of being able to see "oh, `command` is just the command I would be running to start my app, cool!"